_data/templates.json

```diff
@@ -1463,9 +1506,51 @@
 					"Veuillez suivre les instructions d'implémentation sous l'onglet «&nbsp;Standard&nbsp;»\"."
 				]
 			}
+		},
+		{
+			"@id": "_:implement_topic_stackedheader",
+			"iteration": "_:iteration_topic_4",
+			"name": {
+				"en": "Topic page with stacked header",
+				"fr": "Page de sujet avec en-tête superposée"
+			},
+			"introduction": {
+				"en": "Use the stacked main page title to display the page's section title as well as the page title.",
+				"fr": "Utilisez le titre de la page principale superposé pour afficher le titre de la section de la page ainsi que le titre de la page."
+			},
+			"instructions": {
+				"en": [
+					"A full content review needs to be performed. This change cannot be automated.",
+					"Follow the implementation instructions under the \"Standard\" tab."
+				],
+				"fr": [
+					"Un examen complet du contenu doit être effectué. Ce changement ne peut pas être automatisé.",
+					"Veuillez suivre les instructions d'implémentation sous l'onglet «&nbsp;Standard&nbsp;»\"."
+				]
+			}
 		}
 	],
 	"iteration": [
+		{
+			"@id": "_:iteration_topic_4",
+			"name": "Topic page - Iteration 4",
+			"date": "2024-08",
+			"detectableBy": "Stacked main page title to display the page's section title as well as the page title.",
+			"example": {
+				"en": [
+					{
+						"href": "topic-en.html",
+						"text": "Topic page with stacked header"
+					}
+				],
+				"fr": [
+					{
+						"href": "topic-fr.html",
+						"text": "Page de sujet avec en-tête superposée"
```

Comment:
Can you please remove the extra "e" at the end of the following sentence: "Titre du sujet sans en-tête superposée"